app/views/search/find.html.erb

```diff
@@ -1,8 +1,15 @@
-<% @page_title = _("Search") -%>
+<% if @search_term == "" %>
+   <% @page_title = _('openSUSE Software Search') -%>
+<% else if @category != "" && @search_term == "" %>
+      <% @page_title = _("Category:") + " " + (@category.split /(?=[A-Z][a-z])/).join(" ") + " | " + _('openSUSE Software') -%>
+   <% else %>
+      <% @page_title = _("Searching for") + " " + @search_term + " | " + _('openSUSE Software') -%>
+   <% end %>
+<% end %>
 
 <% unless @packagenames.nil? %>
   <%= render :partial => 'search/find_results' %>
 
 <% else %>
```

Comment:
you can remove the empty `<% else %>`, it does not do anything. I know it was not you who wrote it, but we should try to remove dead code to make maintaining it easier.